datanode/dehistory/service_test.go

```diff
@@ -262,6 +272,10 @@ func TestMain(t *testing.M) {
 }
 
 func TestRestoringFromDifferentHeightsWithFullHistory(t *testing.T) {
+	if schemaChanged {
```

Comment:
as discussed, just update the hash, no need for the schemaChanged flag

Reply:
Thanks, my mistake. Fixed https://github.com/vegaprotocol/vega/pull/6702/commits/05f9ce2b83c1e6b5fa485bb137ba0bd202c05e65